components/bl-signature-pad-component/src/buttons/save-png-button.js

```diff
@@ -0,0 +1,37 @@
+import { download } from '../utils/file';
+
+export function SavePNGButton({ signaturePad, component, eventHandlers }) {
```

Comment:
it looks almost the same as the SaveJPGButton component

Reply:
Do you mean it should be one component instead of four?